src/backend/marsha/core/serializers/video.py

```diff
@@ -286,6 +286,117 @@ class InitLiveStateSerializer(serializers.Serializer):
     type = serializers.ChoiceField(LIVE_TYPE_CHOICES)
 
 
+class LiveRegistrationSerializer(serializers.ModelSerializer):
+    """Serializer for liveRegistration model."""
+
+    class Meta:  # noqa
+        model = LiveRegistration
+        fields = (
+            "email",
+            "id",
+            "consumer_site",
+            "lti_user_id",
+            "should_send_reminders",
+            "video",
+        )
+        read_only_fields = (
+            "id",
+            "consumer_site",
+            "lti_user_id",
+            "video",
+        )
+
+    # Make sure video UUID is converted to a string during serialization
+    video = serializers.PrimaryKeyRelatedField(
+        read_only=True, pk_field=serializers.CharField()
+    )
+
+    def validate(self, attrs):
+        """Control or set data with token informations.
+
+        Force the video field to the video of the JWT Token if any.
+        Check email, if present in the token, is equal to the one in the request.
+        Set lti informations if they are present in the token. Control integrity
+        errors and set specific messages.
+
+        Parameters
+        ----------
+        data : dictionary
+            Dictionary of the deserialized values of each field after validation.
+
+        Returns
+        -------
+        dictionary
+            The "data" dictionary is returned after modification.
+
+        """
+        # User here is a video as it comes from the JWT
+        # It is named "user" by convention in the `rest_framework_simplejwt` dependency we use.
+        user = self.context["request"].user
+        video = Video.objects.get(pk=user.id)
```

Comment:
make sure it returns a 404 if this raises DoesNotExist? Maybe you need to use this https://docs.djangoproject.com/fr/2.2/topics/http/shortcuts/#get-object-or-404